study.md

```diff
@@ -152,7 +152,10 @@ What are the 4 most common HTTP verbs used when creating a RESTful API. How
 would you use each?
 
 ```md
-<!-- your answer here -->
+POST: Create new rescources.
+PUT: Update capabilities.
+GET: Read or retrieve a representation rescource.
+PATCH: Modify capabilities.
 ```
```

Comment:
I'm not sure you what you mean by capabilities here but it's more about updating capabilities, but think of it more about modifying the resources involved. 

I think we'll go over this but think of it as CRUD
Create - POST create the resource
Read -  GET get and show me the resource
Update - PATCH  existing resource
Destroy -DELETE the resource